### PR TITLE
Make guide titles action oriented.

### LINKED ISSUE #9: Guide titles should be action oriented
Update guide titles so they reflect the objective of the guide, otherwise they're vague i.e. Hazelcast on Kubernetes could include how to operate it and many other concerns. Same goes about the subtitle - it should indicate what the goal is.

Hazelcast with Spring Boot -> Use Hazelcast in a Spring Boot Application
Subtitle: Create a basic Spring Boot application which can utilize Hazelcast

Hazelcast Client With Quarkus -> Connect To Hazelcast From A Quarkus Application
Subtitle: Create a basic Quarkus application which connects to a Hazelcast cluster.

Hazelcast On Kubernetes -> Deploy Hazelcast on Kubernetes
Subtitle: Learn how to deploy Hazelcast on a Kubernetes environment

etc

### PR DESCRIPTION
**Pull request by enozcan**
Closes #9

### CODE REVIEW

File: home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -41,19 +42,19 @@ Cache your data via Hazelcast in microservices.
 
 
 [.guide]
-=== image:logo-micronaut.png[] xref:caching-micronaut:ROOT:index.adoc[Caching in Micronaut Microservices]
+=== image:logo-micronaut.png[] xref:caching-micronaut:ROOT:index.adoc[Use Hazelcast as Cache in Micronaut Microservices]
```
Comment (cangencer) [inline home/modules/ROOT/pages/index.adoc:45]: "as a Cache"

File: home/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -41,19 +42,19 @@ Cache your data via Hazelcast in microservices.
...
 
-Cache data with Hazelcast in Micronaut.
+Learn how to cache data via Hazelcast in Micronaut applications.
 
 [.guide]
-=== image:logo-springboot.png[] xref:caching-springboot:ROOT:index.adoc[Caching in SpringBoot Microservices]
+=== image:logo-springboot.png[] xref:caching-springboot:ROOT:index.adoc[Use Hazelcast as Cache in SpringBoot Microservices]
```
Comment (cangencer) [inline home/modules/ROOT/pages/index.adoc:50]: "as a Cache"

File: home/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -41,19 +42,19 @@ Cache your data via Hazelcast in microservices.
...
 
-Use Hazelcast as Cache Manager in Spring Boot.
+Learn how to use Hazelcast as Cache Manager in SpringBoot applications.
```
Comment (cangencer) [inline home/modules/ROOT/pages/index.adoc:52]: "Sprint Boot"

File: home/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -96,39 +97,39 @@ Use Hazelcast with Kubernetes.
...
 
-Getting Started with Hazelcast on Kubernetes.
+Learn how to create a Hazelcast cluster on Kubernetes environment.
 
 [.guide]
-=== image:logo-kubernetes.png[] xref:kubernetes-embedded:ROOT:index.adoc[Embedded Hazelcast on Kubernetes]
+=== image:logo-kubernetes.png[] xref:kubernetes-embedded:ROOT:index.adoc[Deploy Hazelcast Applications on Kubernetes]
 
-Getting Started with Embedded Hazelcast on Kubernetes.
+Learn how to deploy an application with embedded Hazelcast on Kubernetes environment.
```
Comment (cangencer) [inline home/modules/ROOT/pages/index.adoc:107]: "on Kubernetes."

File: home/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -96,39 +97,39 @@ Use Hazelcast with Kubernetes.
...
 
-Getting Started with External Hazelcast Client on Kubernetes.
+Learn how to connect an external client to a Hazelcast cluster deployed on Kubernetes environment.
 
 [.guide]
-=== image:logo-kubernetes.png[] xref:kubernetes-sidecar:ROOT:index.adoc[Hazelcast as Sidecar container]
+=== image:logo-kubernetes.png[] xref:kubernetes-sidecar:ROOT:index.adoc[Use Hazelcast as Sidecar Container]
 
-Getting Started with Hazelcast as Sidecar container.
+Learn how to use Hazelcast as a sidecar container for your applications deployed on Kubernetes environment.
```
Comment (cangencer) [inline home/modules/ROOT/pages/index.adoc:117]: "on Kubernetes." (also see in other places)

**Review by cangencer — APPROVED** [review]

**Comment by enozcan** [comment]
Thanks for the review.

**Pull request merged**